apps/web/pages/insights/index.tsx

```diff
@@ -47,7 +46,12 @@ export default function InsightsPage() {
 
   return (
     <div>
-      <ShellMain heading="Insights" subtitle={t("insights_subtitle")}>
+      <Shell
+        withoutMain={false}
+        heading="Insights"
+        subtitle={t("insights_subtitle")}
+        title="Insights"
+        description="View booking insights across your events.">
```

Comment:
@anikdhabal what is the difference between heading and title? description and subtitle?

Reply:
Hey @Udit-takkar Thanks for the review. 
![Screenshot 2024-01-30 142620](https://github.com/calcom/cal.com/assets/81948346/9d850e8f-7e5e-4991-a14c-1fda0682f4d1)
![297650212-bdebd4b7-57ef-40ba-8bd9-94df0d79d0ea](https://github.com/calcom/cal.com/assets/81948346/afe1fb79-fa26-4dc6-901b-f596eb2eb739)

Reply:
We offer the functionality to set the page title and description. Currently, if the page title and description are not explicitly set, the values of the heading and subtitle are used as the default page title and subtitle.